City-of-Helsinki/berth-reservations-ui
# VEN-1228 | Fetching customer profile, with debug panel

## Description :sparkles:

* Fetch customer profile
* Add a debug panel for profile view
* Replace apollo-boost with separate apollo dependencies
* Fix authorization
* Change API endpoint to federation root

## Issues :bug:

### Closes :no_good_woman:

* [VEN-1228](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1228): Fetch customer personal information from the federation

## Testing :alembic:

### Manual testing :construction_worker_man:

1. Log in with a Helsinki-tunnus
2. Navigate to the profile page
3. Look at the profile data and the provided debug panel on the right side of the screen. If you do not have a profile, first go to the Profile UI and create one
4. Return and reload the page
5. Add the service connection, and the page is automatically reloaded
6. Entered profile information should be visible
